Jenkinsfile

```diff
@@ -75,8 +74,8 @@ pipeline {
             }
             post {
                 success {
-                    archiveArtifacts artifacts: '**/target/deegree-webservices-*.war', fingerprint: true
-                    archiveArtifacts artifacts: '**/target/deegree-webservices-handbook*.zip', fingerprint: true
+                    archiveArtifacts artifacts: 'checkout/**/target/deegree-webservices-*.war', fingerprint: true
+                    archiveArtifacts artifacts: 'checkout/**/target/deegree-webservices-handbook*.zip', fingerprint: true
```

Comment:
Are these artifacts still needed considering that we are building an release ?

Reply:
These artefacts are build during release perform step and only then available via Jenkins.